spec.bs

```diff
@@ -1060,22 +1060,28 @@ To <dfn>obtain the plaintext payload</dfn> given an [=aggregatable report=]
 <div algorithm>
 To <dfn>encrypt the payload</dfn> given a [=byte sequence=] |plaintextPayload|,
 public key |pkR| and a [=string=] |sharedInfo|, perform the following steps.
-They return a [=byte sequence=] or an error.
+They either return a [=byte sequence=] or throw an error.
 
 1. Let |info| be the result of [=UTF-8 encoding=] the [=string/concatenation=]
```

Comment:
I'm tempted to assert here that the concatenation will contain only ASCII characters.

Reply:
OK, I added an assertion step. I'll leave this open so we remember to at least file a crbug for our implementation.

Reply:
You know, I think we can drop the assertion. While I believe the sharedInfo string is guaranteed to be ASCII, the spec is about to pass it into a UTF-8 encoder, so it doesn't really matter.

Reply:
Ack